extensions/amp-story/1.0/amp-story-desktop-panels.css

```diff
@@ -89,7 +89,6 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
   transform: scale(0.9) translateX(calc(250% + 192px)) translateY(0%) !important;
   opacity: 0.05 !important;
   transform-origin: left !important;
-  border-radius: 0 !important;
```

Comment:
Do we actually want to clean this up?  Won't this mean publishers can (perhaps accidentally) add a border-radius to pages?

Reply:
Done